src/api/index.js

```diff
@@ -52,7 +52,7 @@ api.interceptors.response.use(
                 }
             }
 
-            if (err.response.status === 403 && err.response.data) {
+            if (err.response.data) {
```

Comment:
Why err contain response, not response contain error?

Reply:
this is response handler function implementation